src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.person;
+
+public abstract class Contact {
+	
+    private String value;
+	
```

Comment:
it would be useful to have a constructor here to create the object and use `super()` in the child classes to call it